src/Components/App/App.tsx

```diff
@@ -33,24 +37,47 @@ const App: React.FC = () => {
     console.log("data", data)
   }, [])
 
-  const keepScore = (continent: keyof Guesses): string  => {
-    console.log('DATA', data)
+  const keepScore = (continent: keyof Guesses): string => {
     const total = correctGuesses[continent] + incorrectGuesses[continent]
-    const score = (correctGuesses[continent]/total * 100).toFixed() + '%'
+    const score = (correctGuesses[continent] / total * 100).toFixed() + '%'
     return score
   }
-
+  const assignSelections = (newSelection: object | string)  => {
+    if (newSelection === 'emoji' || newSelection === 'capitols' || newSelection === 'languages') {
+      setSelectedCategoryApp(newSelection)
+      console.log("CATEGORY", newSelection)
+    } else {
+      setSelectedContinentApp(newSelection)
+      console.log("CONTINENT TYPE", newSelection)
+    }
+  }
   return (
     <main className="app-container">
+
+<!--       <NavLink to='/' className='home-link'>
```

Comment:
Not sure if this is leftover comment syntax or something that should currently be commented out, but for me it's not commented in the IDE and throws an error. Also occurs on line 63, 73, and 78.